WalletWasabi.Gui/UiConfig.cs

```diff
@@ -47,7 +48,11 @@ public UiConfig(string filePath) : base(filePath)
 
 		[DefaultValue(0)]
 		[JsonProperty(PropertyName = "FeeDisplayFormat", DefaultValueHandling = DefaultValueHandling.Populate)]
-		public int FeeDisplayFormat { get; internal set; }
+		public int FeeDisplayFormat
```

Comment:
Replace int with FeeDisplayFormat enum. Create a JSON converter and ensure compatibility. The default value should be FeeDisplayFormat.SatoshiPerByte. 

It should be very similar to what we have right above with WindowState.